sympy/geometry/parabola.py

```diff
@@ -406,3 +407,21 @@ def vertex(self):
             vertex = Point(focus.args[0], focus.args[1] - self.p_parameter)
 
         return vertex
+from sympy import *
+def tangent(a,b,g,f,h,c,x1,y1):  
```

Comment:
You should indent your whole function by 4 spaces for it to be a part of the `Parabola` class. 

Also, the function should not require all these parameters. 
If you want to find the tangent of an existing parabola at any point, it should not require you to explicitly provide every attribute of the parabola to the function, they should be taken from the parent Parabola class. 

The parameters of the function should only be `self` and a `Point`.

Also, you should provide a docstring to explain how the function works, and how it can be used, like the functions above.